core/domain/suggestion_registry_test.py

```diff
@@ -759,8 +759,9 @@ def test_validate_change_question_state_data_schema_version(self):
 
         suggestion.validate()
 
-        suggestion.change.question_dict[
-            'question_state_data_schema_version'] = 0
+        question_dict = suggestion.change.question_dict
+        question_dict['question_state_data_schema_version'] = 0
+        suggestion.change.question_dict = question_dict
```

Comment:
Why do we need to change this? Does it not work as it was coded above?

Reply:
Answered [here](https://github.com/oppia/oppia/pull/6989#discussion_r296733694)